### ISSUE TITLE
Error: The jeep-sqlite element is not present in the DOM when trying to generate migration files via TypeOrm CLI

### ISSUE THREAD

**Issue by KiddoV** [typeorm/typeorm#10653]
### Issue description

I am trying to create migration files with the TypeOrm CLI: ``npx typeorm-ts-node-esm migration:generate src/database/migrations/PostRefactoring -d src/database/dataSource.ts``

### Expected Behavior

Migration files get generated without error.

### Actual Behavior

Got error:
```console
Error: The jeep-sqlite element is not present in the DOM! Please check the @capacitor-community/sqlite documentation for instructions regarding the  web platform.
```

### Steps to reproduce

Using ``capacitor`` as database type:
```ts
export const dataSourceConfig: DataSourceOptions = {
    type: "capacitor",
    driver: new SQLiteConnection(CapacitorSQLite),   // pass the connection wrapper here
    database: databaseName,     // database name without the `.db` suffix
    // mode: "no-encryption",
    entities: entities, //All entity files (tables)
    // entities: ["./entities/*.{js,ts}"],
    migrationsRun: true,
    // dropSchema: true,        // !!!You will lose all data in database if set to `true`
    migrations: [PostRefactoring],
    synchronize: false,         // !!!You will lose all data in database if set to `true`
    logging: "all",
}
```

### My Environment

| Dependency          | Version  |
| ---                 | ---      |
| Operating System    |     Windows 10     |
| Node.js version     |  v20.10.0  |
| Typescript version  | Version 4.9.5  |
| TypeORM version     |   0.3.19  |

### Additional Context

Related issue: https://github.com/capacitor-community/sqlite/issues/508

### Relevant Database Driver(s)

- [ ] aurora-mysql
- [ ] aurora-postgres
- [ ] better-sqlite3
- [ ] cockroachdb
- [ ] cordova
- [ ] expo
- [ ] mongodb
- [ ] mysql
- [ ] nativescript
- [ ] oracle
- [ ] postgres
- [ ] react-native
- [ ] sap
- [ ] spanner
- [X] sqlite
- [ ] sqlite-abstract
- [ ] sqljs
- [ ] sqlserver

### Are you willing to resolve this issue by submitting a Pull Request?

No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

**Comment by eitan-pmw** [comment]
TL;DR - add `await sqlite.initWebStore();` before any database operation (including opening):
```javascript
const sqlite = new SQLiteConnection(CapacitorSQLite);
await sqlite.initWebStore();
```
I was just struggling with it myself. I went into the source code and found the following:
```javascript
/**
     * Checks if the `jeep-sqlite` element is present in the DOM.
     * If it's not in the DOM, this method throws an Error.
     *
     * Attention: This will always fail, if the `intWebStore()` method wasn't called before.
     */
    ensureJeepSqliteIsAvailable() {
        if (this.jeepSqliteElement === null) {
            throw new Error(`The jeep-sqlite element is not present in the DOM! Please check the @capacitor-community/sqlite documentation for instructions regarding the web platform.`);
        }
    }
```

(in source: `@capacitor-community/sqlite/src/web.ts:58; in dist: @capacitor-community/sqlite/dist/esm/web.js)
After fiddling around a bit I realised it's not about calling it in the source, but rather about calling it after initialising the database before connection.
